# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_curl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

A little dirty hacking in order to get the code built with dietlibc ...

  - Rene Rebe <rene@exactcode.de>

--- curl-7.14.0/lib/inet_ntop.c.vanilla	2005-06-16 00:30:20.000000000 +0200
+++ curl-7.14.0/lib/inet_ntop.c	2005-06-16 00:32:20.000000000 +0200
@@ -54,6 +54,9 @@
  *  - uses no statics
  *  - takes a u_char* not an in_addr as input
  */
+
+#define HAVE_INET_NTOA_R_2_ARGS
+
 static char *inet_ntop4 (const u_char *src, char *dst, size_t size)
 {
 #if defined(HAVE_INET_NTOA_R_2_ARGS)
